src/app/app.component.ts

```diff
@@ -82,7 +89,16 @@ export class MyApp {
     // Reset the content nav to have just this page
     // we wouldn't want the back button to show in this scenario
     this.menuCtrl.close().then(() => {
-      this.nav.setRoot(page.component, { 'linkParam': page.param });
+      if (page.component === PushMessagesPage && Object.keys(this.push.pushConfig).length === 0) {
+        this.alert.showAlert(constants.pushAlertMessage, constants.featureNotConfigured, 
+          constants.alertButtons, constants.showDocs, constants.pushDocsUrl);
+          return;
+      } else if (page.component === AuthPage && Object.keys(this.auth.getConfig().length === 0)) {
```

Comment:
why one of them is `getConfig()` and the other one is `.pushConfig`?

Reply:
because there is no method getConfig in push for this SDK

Reply:
yeah, just look at the SDKs and I saw that now